sig-ci/README.md

```diff
@@ -0,0 +1,37 @@
+# SIG: CI
+
+This SIG is focused on CI (continuous integration) experience within Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other Tuesday, 1 PM EST / 10AM PST
```

Comment:
Doesn't this conflict with either the kubernetes sig or the spinnaker as code sig?

Reply:
Yes this is the same time as the biweekly Kubernetes SIG.

Reply:
As long as it was on the opposite week (i.e. starting 1/7), it should be fine, though?

Reply:
Aaah, right you are @plumpy .  Here's the calendar I've been looking at: https://calendar.google.com/calendar/embed?src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc%40group.calendar.google.com&ctz=America%2FLos_Angeles

Reply:
Yep, we will start it in the opposite week from the Kubernetes sig, and we might change the date/time according to the meeting's participates as we go forward.